### PR TITLE
Analysis plan draft

### PR DESCRIPTION
**Pull request by matthew-brett**
Please do give helpful or unhelpful feedback

### CODE REVIEW

File: analysis_plan.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+This is the first analysis plan.
+
+It is provisional.
+
+## Reading to do
+
+* Read about Nipy diagnostics
+* Consider Principal Component Analysis (good tutorial?)
+* Consider Independent Component Analysis (e.g. FSL)
+* Your ideas here.
```
Comment (effigies) [inline analysis_plan.md:12]: I think the Gabrieli lab had some tool about a decade ago...

File: analysis_plan.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+## Analysis plan
+
+* Build simplest possible draft using e.g. standard deviation
+* Review identified scans to see whether they are plausible outliers.
+* Extend with new tools:
+    * Statistical modeling
+    * Motion correction
```
Comment (effigies) [inline analysis_plan.md:20]: Let's reorder these to make them match what we plan to do.

```suggestion
    * Motion correction
    * Statistical modeling
```

File: analysis_plan.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+## Reading to do
+
+* Read about Nipy diagnostics
+* Consider Principal Component Analysis (good tutorial?)
+* Consider Independent Component Analysis (e.g. FSL)
+* Your ideas here.
+
+## Analysis plan
+
+* Build simplest possible draft using e.g. standard deviation
```
Comment (effigies) [inline analysis_plan.md:16]: Good plan. Let's not get too complicated all at once.

**Review by effigies — CHANGES_REQUESTED** [review]
Looks reasonable. Small suggestions.

File: analysis_plan.md (hunk 1 above)
Comment (matthew-brett) [reply analysis_plan.md:12]: Could that be [ArtRepair](https://cibsr.stanford.edu/tools/human-brain-project/artrepair-software.html).

File: analysis_plan.md (hunk 3 above)
Comment (matthew-brett) [reply analysis_plan.md:16]: Thanks!

**Review by effigies — APPROVED** [review]

**Pull request merged**